mutt-alias.sh

```diff
@@ -112,17 +112,18 @@ for directory in "$@"; do
       # then make lower-case
       out_to="$( <<<"$each_to" tr --delete '[:space:]' | sed -E 's/.*<(.*)>/\1/' )"
       out_to=$(echo "$out_to" | tr "[:upper:]" "[:lower:]")
+      name_to="$( <<<"$each_to" tr --delete '[]' | sed --regexp-extended 's/(.*)<.*>/\1/' )"
```

Comment:
I tried to do it without the void `tr ..`, but it did not work as expected. Let me know if you know how to do it.

Reply:
Well, it works fine without now. Never mind.